### PR TITLE
azurerm_communication_service - export primary_connection_string, secondary_connection_string, primary_key, secondary_key

### PR DESCRIPTION
**Pull request by colinbull**
Currently the azure communication services has no way to retrieve the connection strings from the created resource this PR now returns the following attributes upon resource read: 

* primary_connection_string
* secondary_connection_string
* primary_key
* secondary_key

### CODE REVIEW

File: internal/services/communication/communication_service_resource.go [hunk 1]
```diff
@@ -141,6 +141,23 @@ func resourceArmCommunicationServiceRead(d *pluginsdk.ResourceData, meta interfa
...
+			return nil
+		}
+
+		return fmt.Errorf("retrieving %s: %+v", *id, err)
+	}
+
+	d.Set("primary_connection_string", keysResp.PrimaryConnectionString)
+	d.Set("secondary_connection_string", keysResp.SecondaryConnectionString)
+	d.Set("primary_key", keysResp.PrimaryKey)
+	d.Set("secondary_key", keysResp.SecondaryKey)
```
Comment (tombuildsstuff) [inline internal/services/communication/communication_service_resource.go:159]: we also need to add these fields to the schema (above) as Computed strings - and then check that each field is set into the state in a test:

Test: https://github.com/hashicorp/terraform-provider-azurerm/blob/b9ac506edcc02c0ac049cd1f6e85205341c666c9/internal/services/communication/communication_service_resource_test.go#L26

via e.g.

> check.That(data.ResourceName).Key("primary_connection_string").Exists()

**Review by tombuildsstuff — CHANGES_REQUESTED** [review]
hi @colinbull 

Thanks for this PR :)

Taking a look through here whilst this is setting the value into the state, these fields aren't defined in the Schema (at the top of the Resource) and we need to assert that these fields have a value during the tests - however if we can fix those up then we should be able to take another look and get this merged 👍 

Thanks!

**Comment by colinbull** [comment]
@tombuildsstuff Added requested changes above. Let me know if there is anything else I have missed.

**Review by katbyte — APPROVED** [review]
thanks @colinbull - LGTM 🏗️

**Pull request merged**